src/EditorPanel.tsx

```diff
@@ -426,7 +426,7 @@ export const EditorPanel: React.FC<{
           }}
         />
       )}
-      <Box height="100%" display="grid" gridTemplateRows="1fr 4rem">
+      <Box height="100%" display="grid" gridTemplateRows="1fr auto">
```

Comment:
This might not be good if `auto` becomes a different size than the other footer (buttons will be misaligned)